.golangci.yml

```diff
@@ -1,6 +1,7 @@
 ---
 run:
   # concurrency: 6
+  go: "1.17"
```

Comment:
Why is this 1.17 instead of 1.18.1 ?

Reply:
golangci guys is not added support for 1.18. The work in progress see at golangci/golangci-lint#2649.
We'll update this as soon as golangci will be updated.